cmd/jira-lifecycle-plugin/server.go

```diff
@@ -1292,7 +1294,7 @@ func validateBug(bug *jira.Issue, dependents []dependent, options JiraBranchOpti
 		validations = append(validations, fmt.Sprintf("bug %s open, matching expected state (%s)", was, expected))
 	}
 
-	if options.TargetVersion != nil {
+	if checkTargetVersion(options) {
```

Comment:
We probably don't want to allow users to opt-out of target version checks for `OCPBUGS` issues, only non-bug jira references.

Reply:
Good catch!  Reverted.